source/Octopus.Tentacle/Communications/OctopusServerChecker.cs

```diff
@@ -38,34 +38,40 @@ public string CheckServerCommunicationsIsOpen(Uri serverAddress, IWebProxy? prox
 
             Retry("Checking that server communications are open", () =>
             {
-#pragma warning disable DE0003
+                // TODO see if can deal with SYSLIB0014
```

Comment:
Is this TODO still to do? What are the plans for doing it?

Reply:
Removing this TODO. Fix would be to properly inject an `HttpClient` (or the factory) and go async from there. Treating it as out of scope for now, as it might not be a local change. But important to address later on. Ideally make tentacle more async overall.